### ISSUE TITLE
Request: Move most settings to per-user

### ISSUE THREAD

**Issue by caewok** [saif-ellafi/foundryvtt-minimal-ui#31]
Foundry 0.7.9. Minimal UI 0.8.8. 

Right now, only the Border Colors and Shadow Colors are per-user settings, the rest are world-only (GM). Please consider switching nearly all the settings to per-user, up to and including disabling the module altogether on a per-user basis. My thinking here is that different users will have different preferences for how the UI appears. In particular, laptop users versus someone with giant dual monitor setups may have very different preferences. 

If you are interested in more organization for the various settings, check out how the midi-qol module created a Workflow Settings button that pops out a tabbed window. 

Thanks!

**Comment by saif-ellafi** [comment]
Thanks for the feedback. Actually, nothing should be picked by the users. The shadow color and border should not be setteable, it is a bug (it doesn't work anyway).

This was a long discussion, but the reasoning behind it, is that the way I see it, and also some of the feedback I gathered around last months, states that the GM prefers to set up the "pitch" for the players.

One of my GMs for example, set's up a very well defined cyberpunk theme. He chose the theme, the size of the elements and also the behaviour. This way also, when players get stuck, both the GM and the players all see the same thing.
Having players set up customized things would be either: not overall used (honestly I think player's don't really care), and second, it would bring only chaos and despair :) with things like "where's my macro?" - "how do you see it?" kind of things.

So at the moment, it is kind of a design choice that the GM set's up the interface for his game.

This said, I'd still be open to consideration with -some- of the settings, like controls size (different screen, different devices), but not much more than that really, for the reasons above.

**Comment by caewok** [comment]
Fair enough. I suppose I was thinking more of the physical modifications and less about the visual ones. The switch to having UI disappear until you move the mouse to hover over an area is a pretty big change. The change to the size of the canvas controls bar is another. I think they are great, but I definitely have players that don't do well with change! Plus the different screen size / different devices issue. So there is probably room for a good middle ground here.

**Comment by saif-ellafi** [comment]
Thanks, yes. I wanted to leave the consideration open, if you don't mind. A middle ground can be something achievable as you said.

However, in regards of screen size, many changes in Minimal UI take in consideration user's display. The objective behind is that more or less players should relatively see the same thing. Up to some extent.

If you don't mind I'll leave the issue open, the won't fix label is also wrong. Could be misunderstood.

**Comment by marcstraube** [comment]
Hi! I found this module just yesterday and I really like how it optimizes Foundry's UI for working on a 13" laptop.
Maybe it would be an option, to just save the users "pinned" settings, so that the side- and macrobar stay pinned after reloading the page.

**Comment by Bryn1701** [comment]
I also just found this module and I really like it. But I know some of my players are going to want control over their bits and pieces. I would love to also see an ability if they lock their bars to stop from auto hiding it will stick after a refresh. A good compromise I think between full control and no control. But I don't code so I don't know if it would be a nightmare or not.

**Comment by JohnTheLate** [comment]
Commenting this from a player perspective: I'd be great to have per-player options. Ideally, there'd be no need to use them, but there might be some where the playerbase disagrees about the interface. In my case, I don't like how I have no way to keep the player list in normal display mode when the host configured it differently, thus having to click it every time I want to see who dropped or is online or missing or who is what colour again.
